docker/native-linux-runner

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# shellcheck disable=SC1091
+. lib.sh
+
+main() {
```

Comment:
We might actually be able to default to the native cross runner here, since this will always be a native runner:

```bash
if [ -Z "${CROSS_RUNNER}" = "" ]; then
   export CROSS_RUNNER=native
fi
```